spec/requests/corporate_information_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe "Corporate Information Page" do
+  before do
+    content_store_has_example_item("/government/organisations/government-digital-service/about", schema: :corporate_information_page)
+    content_store_has_example_item("/government/organisations/government-digital-service/about/our-governance", schema: :corporate_information_page)
```

Comment:
Why does the "our-governance" page need to be stubbed?

Reply:
I had to configure both `/about` and `/about/:slug` to render the show template in routes.rb so I though I should test both routes here.